db/migrate/20151106172531_remove_uniqueness_display_name_index_users.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniquenessDisplayNameIndexUsers < ActiveRecord::Migration
+  def change
+    remove_index :users, column: :display_name, unique: true
```

Comment:
@brian-c @aweiksnar @parrish does the front end require unique display_names anything (filtering?) these days? Pretty sure the API uniquely filters on slugs and owners only.